lke_clusters_acl.go

```diff
@@ -0,0 +1,68 @@
+package linodego
+
+import "context"
+
+// LKEClusterControlPlaneACLAddresses describes the
+// allowed IP ranges for an LKE cluster's control plane.
+type LKEClusterControlPlaneACLAddresses struct {
+	IPv4 []string `json:"ipv4,omitempty"`
+	IPv6 []string `json:"ipv6,omitempty"`
+}
+
+// LKEClusterControlPlaneACL describes the ACL configuration
+// for an LKE cluster's control plane.
+type LKEClusterControlPlaneACL struct {
+	Enabled   *bool                               `json:"enabled,omitempty"`
+	Addresses *LKEClusterControlPlaneACLAddresses `json:"addresses,omitempty"`
+}
+
+// LKEClusterControlPlaneACLUpdateOptions represents the options
+// available when updating the ACL configuration of an LKE cluster's
+// control plane.
+type LKEClusterControlPlaneACLUpdateOptions struct {
+	ACL LKEClusterControlPlaneACL `json:"acl"`
```

Comment:
The structure of this request body deviates a bit from what you'd expect but I did confirm that this is what the API accepts